app/javascript/components/chart/ChartResults.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="chart--results">
+    <svg width="100%" height="100%" :viewBox="`0 0 ${options.svgWidth} ${options.svgHeight}`" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid">
+      <rect x="0" y="0" width="100%" height="100%" fill="#ffffff" stroke="#dedede" stroke-width="2"></rect>
+
+      <g v-for="axis, index in axes"> 
+        <text :x="options.axisStart - 35" :y="config.offsetTop + 3 + index*100" text-anchor="end" fill="#466882" font-weight="bold">{{ axis }}</text>
+        <text :x="options.axisStart - 10" :y="config.offsetTop + 3 + index*100" text-anchor="end" font-size="12">{{ domain[0] }}</text>
+        <line  :x1="options.axisStart" :y1="config.offsetTop + index*100" :x2="options.axisEnd" :y2="config.offsetTop + index*100" stroke="#466882"/>
+        <text :x="options.axisEnd + 10" :y="config.offsetTop + 3 + index*100" font-size="12">{{ domain[1] }}</text>
+      </g>
+      
+      <g v-for="line, index in data">
+        <path
+          :d="getPath(line.dataset)" 
+          fill="none" 
+          :stroke="getColour(line.current_user)" 
+          :stroke-width="getWidth(line.current_user)" />
+        
+        <circle v-if="line.current_user" v-for="datapoint, index in line.dataset" :cx="getXValue(datapoint) + options.axisStart" :cy="config.offsetTop + index*100" r="8" :fill="config.colours.currentUser"></circle>          
+        <circle v-for="datapoint, index in line.dataset" :cx="getXValue(datapoint) + options.axisStart" :cy="config.offsetTop + index*100" r="4" :fill="config.colours.default"></circle>
+      </g>
+    </svg>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'chart-results',
+
+    props: {
+      data: {
+        type: Array,
+        required: true
+      }
+    },
+
+    data () {
+      return {
+        options: {
+          svgWidth: 600,
+          svgHeight: 330,
+          axisStart: 160,
+          axisEnd: 510
+        },
+        config: {
+          offsetTop: 0,
+          colours: {
+            default: '#59A8D0',
+            currentUser: '#466882'
+          },
+        },
+        domain: [-4, 4],
+        range: [0],
+        axes: ['Capitalism', 'People', 'Nature']
```

Comment:
Are the labels not being letters a change in design or just not done yet? Also in the designs there isn't the classification bit under the graph.